### PR TITLE
Add a compatibility matrix in README.

### PR DESCRIPTION
**Pull request by qingling128**

### CODE REVIEW

**Comment by rmoriar1** [comment]
Looks like the validation command `/opt/google-cloud-ops-agent/subagents/collectd/sbin/collectd -tC` is failing for the ops-agent was the path changed?

Also appears sles-15 is failing on restart.

**Comment by jsirianni** [comment]
> Looks like the validation command `/opt/google-cloud-ops-agent/subagents/collectd/sbin/collectd -tC` is failing for the ops-agent was the path changed?
> 
> Also appears sles-15 is failing on restart.

I do not have access to the CI so I am unsure what this check is for, but I think it can be changed to: 
```
ops-agent_validation_cmd: '/opt/google-cloud-ops-agent/subagents/fluent-bit/bin/fluent-bit --version'
```

My assumption is that it is just a smoke test to ensure the agent installed a binary where it is expected?

**Comment by rmoriar1** [comment]
> My assumption is that it is just a smoke test to ensure the agent installed a binary where it is expected?

It's to ensure that the custom configuration is valid. I think the new syntax should be: 

`ops-agent_validation_cmd: '/opt/google-cloud-ops-agent/subagents/fluent-bit/bin/fluent-bit -c %s'`

**Comment by quentinmit** [comment]
> > My assumption is that it is just a smoke test to ensure the agent installed a binary where it is expected?
> 
> It's to ensure that the custom configuration is valid. I think the new syntax should be:
> 
> `ops-agent_validation_cmd: '/opt/google-cloud-ops-agent/subagents/fluent-bit/bin/fluent-bit -c %s'`

That doesn't work for Ops Agent; Ops Agent uses a single config file for both logging and monitoring, and that config file is not read by any of the subagents.

Also we're about to move the subagents into a new directory; you should not rely on anything in /opt/google-cloud-ops-agent to be present or have any particular functionality.

**Comment by jsirianni** [comment]
I think that is okay. Does the ops agent have the ability to validate the configuration at `/etc/google-cloud-ops-agent/config.yaml`?

**Comment by quentinmit** [comment]
There is not currently a supported way to do that, no. It's a good feature request.

File: README.md [hunk 1]
```diff
@@ -124,6 +124,20 @@ Example Playbooks
...
 
+
+Compatibility
+-------------
+
+The matrix below lists the versions of this Ansible role and the agent versions it supports.
+
+| Ansible Role Version | Compatible Ops Agent Version(s) | Compatible Logging Agent Version(s) | Compatible Monitoring Agent Version(s) |
+|----------------------|-------------------------------- | ----------------------------------- | -------------------------------------- |
+| **1.x.x**            | 1.x.x                           | 1.x.x                               | 6.x.x                                  |
```
Comment (rmoriar1) [inline README.md:135]: Ops agent can be bumped to 2.x.x once #74 is checked in.

**Review by rmoriar1 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (qingling128) [reply README.md:135]: Yup!

**Pull request merged**